src/containers/main-page/cv-form/components/fields/certifications/components/addedSertificates/AddedCertificatesItem.tsx

```diff
@@ -0,0 +1,25 @@
+import { memo } from 'react';
+
+import ProfiencyItem from 'common/components/profiency/ProfiencyItem';
+
+import { useDeleteUserCertificate } from '../../lib/query-hooks';
+
+const AddedCertificatesItem = function ({
+  certificate, id, link,
+}: {certificate: string, id: Date | null | string, link: string | undefined}): JSX.Element {
```

Comment:
Please replace `{certificate: string, id: Date | null | string, link: string | undefined}` to interface.